applications/callflow/src/module/cf_pivot.erl

```diff
@@ -52,6 +52,15 @@ handle(Data, Call) ->
     lager:info("published pivot request"),
     wait_for_pivot(Data, Call).
 
+-spec maybe_use_variable(wh_json:object(), whapps_call:call()) -> api_binary().
+maybe_use_variable(Data, Call) ->
```

Comment:
This function name gives me no indication of what the function actually does. I'm generally confused as to what's going on and why this functionality is needed. Perhaps you could write up some docs in `callflow/doc` to describe what you're trying to accomplish with this function and the `cf_kvs_set` module?